plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -165,4 +165,7 @@ private MLCommonsSettings() {}
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         );
+    
+    public static final Setting<Float> ML_COMMONS_DISK_SHORTAGE_THRESHOLD = Setting
+        .floatSetting("plugins.ml_commons.disk_shortage_threshold", 5L, 10L, 1L, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
Can we have a longSetting here? Why the min value is 10L and the max value is 1L for this setting? Did you have a successful test on this setting change on your local cluster? Or can we have a test case to cover it?